src/azure-cli/azure/cli/command_modules/security/tests/latest/recordings/test_security_atp_settings.yaml

```diff
@@ -0,0 +1,265 @@
+interactions:
+- request:
+    body: null
+    headers:
+      Accept:
+      - application/json
+      Accept-Encoding:
+      - gzip, deflate
+      CommandName:
+      - security atp show
+      Connection:
+      - keep-alive
+      ParameterSetName:
+      - --resource-group --storage-account-name
+      User-Agent:
+      - python/3.8.0 (Windows-10-10.0.18362-SP0) msrest/0.6.10 msrest_azure/0.6.2
+        azure-mgmt-security/0.1.0 Azure-SDK-For-Python AZURECLI/2.0.77
+      accept-language:
+      - en-US
+    method: GET
+    uri: https://management.azure.com/subscriptions%2F291f4b4b-4316-421b-80d0-6042e950e8c5%2FresourceGroups%2Fclitest.rg000001%2Fproviders%2FMicrosoft.Storage%2FstorageAccounts%2Fclitest000002/providers/Microsoft.Security/advancedThreatProtectionSettings/current?api-version=2017-08-01-preview
+  response:
+    body:
+      string: '{"properties":{"isEnabled":false},"id":"/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/clitest.rg000001/providers/Microsoft.Storage/storageAccounts/clitest000002/providers/Microsoft.Security/advancedThreatProtectionSettings/current","name":"current","type":"Microsoft.Security/advancedThreatProtectionSettings"}'
+    headers:
+      cache-control:
+      - no-cache
+      content-length:
+      - '401'
```

Comment:
I think `Content-Length` is an integer, not a string.
Not critical though.

Reply:
This is a recorded file generated by [pyvcr](https://github.com/kevin1024/vcrpy) automatically.